test-scripts/resourceOwnerPasswordGrant.py

```diff
@@ -71,11 +69,11 @@ class OAuthConfig(Config):
 client.prepare_request_body(username=username, password=password, scope=['ci.write', 'ci.read'])
 oauth = OAuth2Session(client=client)
 
-print " OAuth2 Session has been created for client: {}".format(client_id)
+print(" OAuth2 Session has been created for client: {}".format(client_id))
 
 token = oauth.fetch_token(token_url='http://localhost:8000/api/v1/tokens/', username=username, password=password, client_id=client_id, client_secret=client_secret)
 
-print " *** Access Token Granted *** "
-print " Values are: "
+print(" *** Access Token Granted *** ")
```

Comment:
Same for all the 'print' lines here to. can we make them all logger statements?
thank you in advance.